# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/svg/SVGClipPathFrame.cpp
# Commit: b16351df250c
# Full Hash: b16351df250c572a14efa126f7702fe104ac9fd7
# Author: Robert Longson <longsonr@gmail.com>
# Date: 2023-07-22 21:26:48
# Regressor Bug: 1843945
# File Overlap Count: 1
# Description:
#   Bug 1843945 - optimise SVGObserverUtils r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D183802
# ==============================================================================

diff -r 7e8063ebfd83 -r b16351df250c layout/svg/SVGClipPathFrame.cpp
--- a/layout/svg/SVGClipPathFrame.cpp	Sat Jul 22 10:03:30 2023 +0000
+++ b/layout/svg/SVGClipPathFrame.cpp	Sat Jul 22 14:06:38 2023 +0000
@@ -376,12 +376,12 @@
                                             int32_t aModType) {
   if (aNameSpaceID == kNameSpaceID_None) {
     if (aAttribute == nsGkAtoms::transform) {
-      SVGObserverUtils::InvalidateDirectRenderingObservers(this);
+      SVGObserverUtils::InvalidateRenderingObservers(this);
       SVGUtils::NotifyChildrenOfSVGChange(
           this, ISVGDisplayableFrame::TRANSFORM_CHANGED);
     }
     if (aAttribute == nsGkAtoms::clipPathUnits) {
-      SVGObserverUtils::InvalidateDirectRenderingObservers(this);
+      SVGObserverUtils::InvalidateRenderingObservers(this);
     }
   }
 